src/libs/actions/Task.js

```diff
@@ -125,10 +125,96 @@ function createTaskAndNavigate(currentUserEmail, parentReportID, title, descript
     Navigation.navigate(ROUTES.getReportRoute(optimisticTaskReport.reportID));
 }
 
+/**
+ * Sets the task editing property in the store to true
+ *
+ * @params {Object} report
+ */
+
+function beginEditingTask() {
+    Onyx.merge(ONYXKEYS.TASK, {isEditing: true});
+}
+
+/** Sets the task editing property in the store to false */
+function endEditingTask() {
+    Onyx.merge(ONYXKEYS.TASK, {isEditing: false});
+}
```

Comment:
How come we need to know if we are currently editing a task or not? There is another form we could use that we are currently using for title / description which is EDIT_TASK_FORM

Reply:
This is for the `TaskAssigneeSelectorModal` I'm reusing it and want it to _effectively_ do the same thing, just call a different function if we're editing / creating a new task. Alternatively I could instead do a check to see if a report exists on the `TASK` onyx key, which I'm setting whenever we're on a task report page.

Reply:
Gotcha, yeah this approach is probably better

>  Alternatively I could instead do a check to see if a report exists on the TASK onyx key, which I'm setting whenever we're on a task report page.

This might not work since the task won't always be stored in onyx